package.json

```diff
@@ -72,8 +72,12 @@
         "react": "^17.0.1",
         "react-dom": "^17.0.1",
         "react-router-dom": "^5.2.0",
+        "react-switch": "^6.0.0",
```

Comment:
lets use vanilla bootstrap switch instead, 50kb is to much for a switch

https://getbootstrap.com/docs/5.0/forms/checks-radios/#switches

Reply:
make sense!